Quoridor.Model/Graph.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Collections.Generic;
+
+namespace Quoridor.Model {
+    class Graph {
+        private int size;
```

Comment:
[private fields in C# starts with **_**.](https://docs.microsoft.com/ru-ru/dotnet/csharp/fundamentals/coding-style/coding-conventions#camel-case)
It will help You in constructors and will reduce **"this."** in code